var/spack/repos/builtin/packages/legion/package.py

```diff
@@ -277,6 +279,9 @@ def validate_gasnet_root(value):
         default=1024,
         description="Maximum number of nodes supported by Legion.",
     )
+    variant("prof", default=False, description="Install Rust Legion prof")
+
+    depends_on("rust", type="build", when="+prof")
```

Comment:
Our profiler depends on at least Rust 1.74. Is that something we can assume here?

Reply:
I can add that, currently it supports up to 1.76

Reply:
Updated.